epcis/src/main/java/io/openepcis/model/epcis/util/DefaultJsonSchemaNamespaceURIResolver.java

```diff
@@ -18,6 +18,7 @@
 import java.util.HashMap;
 import java.util.Map;
 import java.util.Optional;
+import java.util.concurrent.ConcurrentHashMap;
 
 public class DefaultJsonSchemaNamespaceURIResolver {
```

Comment:
still using shared  
```
private static DefaultJsonSchemaNamespaceURIResolver context;
``` 
context instance is shared across all threads and will eventually grow over time. please make sure to cover with proper unit test